main.py

```diff
@@ -1,12 +1,197 @@
+            print_stars()
+            for video in video_store.get_videos_info():
+                # print(video)
+                print(f"ID: {video['id']} - Title: {video['title']}")
+
+        elif choice == '2':
+            print("Great! Let's add a new video.")
+            title=input("What is the title of your video? ")
+            release_date=input("What is the release date of your video? (MM/DD/YYYY): ")
+            total_inventory=input("What is the total inventory? ")
+            response = video_store.add_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print_stars()
+            print("New video: ") 
+            print(response)
+            print_stars()
+
+        elif choice == '3':
+            print("Fantastic! Let's choose a video!")
+            video_id = input("Enter a video id: ")
+            response = video_store.get_video_info(video_id)
+            print_video(response)
+
+        elif choice == '4':
+            print("Great! Let's update the video!")
+            video_id = input("Enter a video id: ")
+            title=input("What is the new title of your video? ")
+            release_date=input("What is the new release date of your video? (MM/DD/YYYY): ")
+            total_inventory=input("What is the new total inventory? ")
+            response = video_store.edit_video(video_id=video_id, title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_stars()
+            print("Updated video: ")
+            print(response)
+            print_stars()
+
+        elif choice == '5':
+            print("Which video would you like to delete?")
+            video_id = input("Enter a video id: ")
+            video_store.delete_video(video_id)
+            print_stars()
+            print("Video has been deleted.")
+            print_stars()
+
+        elif choice == '6':
+            print_stars()
+            for customer in video_store.get_customers_info():
+                # print(customer)
+                print(f"ID: {customer['id']} - {customer['name']} - Phone: {customer['phone']} - Rented Videos: {customer['videos_checked_out_count']}")
+            print_stars()
+
+        elif choice == '7':
+            print("Wonderful! Let's add a new customer!")
+            name=input("What is the name of the customer? ")
+            postal_code=input("What is the postal code of the customer? ")
+            phone=input("What is the phone number of the customer? ")
+            response = video_store.add_customer(name=name, postal_code=postal_code, phone=phone)
+            print_stars()
+            print("New customer: ") 
+            print(response)
+            print_stars()
+
+        elif choice == '8':
+            print("Whohoo! Let's choose a customer!")
+            customer_id = input("Enter a customer id: ")
+            response = video_store.get_customer_info(customer_id)
+            print_customer(response)
```

Comment:
Similar to getting a video, this line raises an exception for a customer that does not exist. Consider how to handle a 404 response.